### PR TITLE
Fix :  Don't parse onchange resullt when returning  False instead of nothing

### PR DESCRIPTION
**Pull request by flotho**
In case of onchange method calling method that retrun possible false values https://github.com/OCA/OCB/blob/10.0/addons/account/models/partner.py#L162 the syntax value[0] failed because of the structure fo the dictionnary : 
values = {'fiscal_position_id': False}
This patch prevent to pass in those lines

### CODE REVIEW

**Comment by flotho** [comment]
@pedrobaeza , how is it possible to propagate this on later release?
Any specific command to add ?

**Comment by pedrobaeza** [comment]
Do you mean forward-port it?

**Comment by flotho** [comment]
> Do you mean forward-port it?

yes

**Comment by pedrobaeza** [comment]
Manually cherry-picking the commit and proposing/committing (if you are maintainer) it, or try to use this tool:

https://github.com/OCA/maintainer-tools/pull/504

**Comment by flotho** [comment]
Hi @pedrobaeza 

Regarding runbot, what can I do?

BTW, I'm waiting everything green to squash my commits ;-)

**Comment by pedrobaeza** [comment]
Forcing rebuild can lead to get a proper one. I have launched it to see.

**Comment by flotho** [comment]
Hi @pedrobaeza thanks for your help.
I can't see anything in the logs  and I'm not able to restart the build 
image

sorry for those boring technical questions ;-)

**Comment by pedrobaeza** [comment]
No need to launch again runbot. The logs are there: https://runbot2-3.odoo-community.org/runbot/static/build/3497520-61-5b4a29/logs/job_20_test_all.txt

and these are the warnings:
```
2021-08-10 18:41:48,753 184 WARNING openerp_test odoo.models: exception.rule.create() includes unknown fields: rule_group
2021-08-10 18:41:48,760 184 WARNING openerp_test odoo.models: exception.rule.create() includes unknown fields: rule_group
2021-08-10 18:41:48,765 184 WARNING openerp_test odoo.models: exception.rule.create() includes unknown fields: rule_group
```

**Comment by flotho** [comment]
Thanks @pedrobaeza ,

In the end, it looks like I already have this issue https://github.com/OCA/sale-workflow/issues/1058

**Comment by flotho** [comment]
Hi there,

everything looks green, could we merge ?

**Comment by flotho** [comment]
any chance to be merged @pedrobaeza  ?
Regards

**Comment by pedrobaeza** [comment]
This requires at least 2 reviews from qualified people.

**Comment by flotho** [comment]
> This requires at least 2 reviews from qualified people.

(facepalm) you're right, holidays were too short!

how can I identify maintainers of this repo?

**Comment by pedrobaeza** [comment]
Check previous committers of the module, but it also helps and it's fair that you review other PRs in exchange.

**Comment by flotho** [comment]
Hi @lmignon @guewen 

found your contribution in the history.
Any review possible here ?

Regards

**Review by lmignon — APPROVED** [review]
LGTM (code review)

**Review by guewen — APPROVED** [review]

**Pull request merged**
